### DISCUSSION TITLE
对于各不相同的onebot实现，是否可以考虑加一个标准API：api_available

### DISCUSSION THREAD

**Discussion by crazywhalecc** [howmanybots/onebot#79] [想法 / 建议]
这个是用来检查其他API是否可用的一个API，如果标准里面有这个，就可以在onebot的对接SDK上调用这个来确定API是否可用了？要不然每次用404来试感觉不优雅。而且有些如果要是对接其他平台，比如微信公众号，就不能使用set_group_admin之类的API，那个onebot兼容层实现起来就需要对那些接口返回不可用的状态。

**Comment by stdrc** [comment]
也许这个可以在群里讨论一下看看各实现作者的意见

**Comment by yyuueexxiinngg** [comment]
还有支持的消息段列表也可以考虑

**Comment by stdrc** [comment]
对每个 API/消息段，可以返回一个支持的级别：`full` `partial`，比如有些实现也许只支持了部分参数，更好区分

**Comment by crazywhalecc** [comment]
这个计划要打算实现了吗
